about.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Green Earth Summit</title>
+</head>
+<body>
+    <nav>
+        <div class="upper-nav">
+            <ul class="top">
+                <li class="fb"><img  id="fb" src="assets/fb (1).png" alt="facebook icon"/></li>
+                <li class="twitter"><img id="twitter" src="assets/twitter.png" alt="twitter icon"></li>
+                <li class="pa"><p>English</p></li>
+                <li class="pa"><p>MyPage</p></li>
+                <li class="pa"><p>Logout</p></li>
+            </ul>
+        </div>
+        <div class="hamburger">
+            <ul>
+            <li class="bar"></li>
+            <li class="bar"></li>
+            <li class="bar"></li>
+        </ul>
+        </div>
+        <div  class="flex" >
+          <a  class="logo" href="index.html"><img  id="logo"  src="assets/Earth.png"/></a>
+          <ul class="nav-menu">
+            <li class="nav-item">
+            <a class="nav-link" id="nav1"  href="index.html">Home</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Program</a>
+          </li>
+          <li class="nav-item">
+              <a class="nav-link" href="#">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Sponsor</a>
+            </li>
+          <li class="nav-item">
+            <a class="nav-link"  href="#">News</a>
+          </li>         
+          <li class="nav-item GE">
+              <a class="nav-link" id="ge" href="#">Campaign</a>
+            </li>    
+        </ul>
+        </div>
+    </nav>
+   <!--Header-->
+   <header>
+    <h1 class="heading1about">"Hello!Sharing world"</h1>
+    <h2 class="heading2">Green Earth Global Summit 2023</h2>
```

Comment:
- [ ] I suggest to consider switching the heading tags. If you look at the second heading, it it the one that describes the overview of the application. Therefore to increase the SEO of the application, the h2 tag should be changed to h1 ☺️.